lib/Transforms/ElementwiseToAffine/ElementwiseToAffine.td

```diff
@@ -13,6 +13,12 @@ def ElementwiseToAffine : Pass<"convert-elementwise-to-affine"> {
     "mlir::affine::AffineDialect",
     "mlir::tensor::TensorDialect"
   ];
+  let options = [
```

Comment:
could you please add a quick comment in the pass description to describe how to use these options? e.g. a user can *either* set dialects or ops (it's not necessary to to dialects=polynomial if using ops=polynomial.add)

Reply:
From the semantic side, I am thinking
`'restric-ops=polynomial.add,polynomial.mul_scalar restrict-dialects=polynomial'`   would allow all instructions in polynomial dialect. 
And 
` 'restrict-ops=polynomial.add,polynomial.mul_scalar restrict-dialects=bgv'`
in the above case it will allow two ops from polynomial case and all ops from bgv.

Reply:
I agree that makes sense to me! Especially the second example you had would be a good example to add to the description

Reply:
Sure